server/src/main/java/org/opentosca/toscana/plugins/cloudformation/util/CloudFormationScript.java

```diff
@@ -10,11 +10,16 @@
 
 import static org.opentosca.toscana.plugins.cloudformation.CloudFormationModule.FILEPATH_TARGET;
 
+/**
+ Class that overrides the {@link BashScript} implementation.
+ */
 public class CloudFormationScript extends BashScript {
 
     private final static Logger logger = LoggerFactory.getLogger(BashScript.class);
 
     /**
+     Standard constructor.
```

Comment:
I don't get how the  information `Standard constructor` provides any benefits. Could you explain?